bitbucket-slack-server-integration-plugin/src/main/resources/com/atlassian/bitbucket/plugins/slack/bitbucket-slack.properties

```diff
@@ -103,6 +103,7 @@ bitbucket.plugins.slack.notification.PRReopened=Reopened
 bitbucket.plugins.slack.notification.PRDeleted=Deleted
 bitbucket.plugins.slack.notification.PRMerged=Merged
 bitbucket.plugins.slack.notification.PRUpdated=Updated
+bitbucket.plugins.slack.notification.PRReScoped=Updated
```

Comment:
I believe this isn't necessary to add this label here as we don't plan to add this event to the repository subscription page - to `bitbucket.slack.feature.repoChannelTable.renderCategoryItemList` template.